Source/Core/Resource.js

```diff
@@ -1000,6 +1009,7 @@ function fetchImage(options) {
       crossOrigin,
       deferred,
       flipY,
+      skipColorSpaceConversion,
```

Comment:
I did a global search for `createImage` and noticed that some terrain and imagery tests require updates. See `ArcGisMapServerImageryProviderSpec.js` and others examples like below - the new parameter needs to be added.

```
          Resource._DefaultImplementations.createImage(
            request,
            crossOrigin,
            deferred,
            true,
            true
          );
```